syntax = "proto3";



message AccountSimplePlayer
{
  uint64 player_id = 1;
}

message AccountSimplePlayerList
{
  repeated AccountSimplePlayer players = 1;
}
